CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 
+## 5.2.3 - 2020-09-29
+### Changed
+- Increased DX (Developer Experience) and modified output of DumpRoleHierarchyCommand.
+
+Before
+```
+'app/console' '--env=development' 'zicht:admin:dump-role-hierarchy' 'DUPLICATE' 'TRANSLATE' '--root=ROLE_ROOT'
+```
+
+After
+```
+app/console --env=development zicht:admin:dump-role-hierarchy DUPLICATE TRANSLATE --root=ROLE_ROOT
```

Comment:
:heart: Wel heel blij met de aanpassing! :smile_cat:

Reply:
Het is mij een beetje om het even. Goed dat er mensen blij mee zijn :smile: